src/System.Private.Uri/tests/System.Private.Uri.Tests.builds

```diff
@@ -16,6 +16,16 @@
       <TestTFMs>netcore50;netcoreapp1.0;net46</TestTFMs>
       <OSGroup>Windows_NT</OSGroup>
     </Project>
+    <Project Include="ExtendedFunctionalTests\System.Private.Uri.ExtendedFunctional.Tests.csproj">
+      <TargetGroup>netstandard1.7</TargetGroup>
+      <SkipTestRun>true</SkipTestRun>
```

Comment:
Yup, changed.

---
In reply to: [72678764](https://github.com/dotnet/corefx/pull/10297#discussion_r72678764) [](ancestors = 72678764)